components/navbar/ContributeButton.tsx

```diff
@@ -0,0 +1,21 @@
+"use client";
```

Comment:
Why is this a `use client` component?

Reply:
sorry for this silly mistake, I am removing this line from this component